CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Add validation for links under the `doc` role.
```

Comment:
This is a bit anal, but blank line after `### Added`, and ticket number in parenthesis right before the period. e.g.

```
### Added

- Add validation for `doc` role targets (DOCSP-XXXX)
```